app-office/abiword/files/abiword-3.0.5-musl-lose-precision-fix.patch

```diff
@@ -0,0 +1,15 @@
+# Patch taken from Alpine linux
+# https://git.alpinelinux.org/aports/tree/community/abiword/musl-1.2.3.patch
```

Comment:
In the Gentoo bug, I give an upstream link too, so please include that here as well.

Reply:
Done